src/main/java/org/broadinstitute/hellbender/tools/validation/compare-with-gatk4.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
This name is not the best (`compare-with-gatk4.sh`) -- how about `validate-reads-pipeline.sh`?